src/internal/task/task_stack_cortexm.go

```diff
@@ -0,0 +1,76 @@
+// +build scheduler.tasks, cortexm
+
+package task
+
+import "unsafe"
+
+const stackSize = 1024
+
+// calleeSavedRegs is the list of registers that must be saved and restored when
+// switching between tasks. Also see scheduler_cortexm.S that relies on the
+// exact layout of this struct.
+type calleeSavedRegs struct {
+	r4  uintptr
+	r5  uintptr
+	r6  uintptr
+	r7  uintptr
+	r8  uintptr
+	r9  uintptr
+	r10 uintptr
+	r11 uintptr
+
```

Comment:
was this newline intentional?

Reply:
Yes. It makes it more readable, because otherwise `pc` would look like just another GPR.